docs/source/host-collect-analyze/dns.md

```diff
@@ -3,109 +3,151 @@ title: Host DNS
 description: Collect DNS queries on the host machine.
 ---
 
-The `dns` host collector can be used to help diagnose DNS resolution problems on the host machine. During execution, the collector performs various DNS `A` record queries to troubleshoot DNS resolution. It does the following:
+The `dns` host collector can be used to help diagnose DNS resolution problems on the host machine. During execution, the collector performs various DNS record queries to troubleshoot DNS resolution. It does the following:
 
 - Reads the contents of `/etc/resolv.conf`
-- Performs DNS lookups for specified hostnames
-- Outputs query results including IP addresses and the result of the DNS lookup
+- Performs DNS `A`, `AAAA`, `CNAME` lookups for specified hostnames
+- Outputs query results including IP addresses (if any)
 
 ## Parameters
 
 In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `dns` host collector accepts the following parameters:
 
-##### `hostnames` (Optional)
+##### `collectorName` (Required)
```

Comment:
this is optional, isn't it?  It's a common parameter that is optional, but best to use.  What happens if we don't supply it, we just get the output in /host-collectors/dns/result.json?

Reply:
we've talked, this is good